mobile-app/cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts

```diff
@@ -33,7 +33,7 @@ context('Wallet - Transaction Authorization with Error', () => {
 
   it('should not reset attempts on cancel', function () {
     cy.getByTestID('button_confirm_convert').click().wait(2000)
-    Array.from(Array(MAX_PASSCODE_ATTEMPT - 1), (v, i) => i + 1).forEach(() => {
+    cy.wrap(Array(MAX_PASSCODE_ATTEMPT - 1)).each(() => {
```

Comment:
Curious, what was the problem if we keep `Array.from...`?

Reply:
Foreach is async function so I have replaced it by cypress loop

Reply:
nice catch, didnt know `forEach()` in cypress is running in parallel and not sequential, thanks!